### PR TITLE
[ Portfolio: mobile menu ] Microverse Project

### PR DESCRIPTION
**Pull request by mr-abdellah**
In this project we added a dynamic menu , it open and close when the user click on the hamburger menu icon and if the user follow a link on it also

### CODE REVIEW

**Comment by mr-abdellah** [comment]
hello, please , I got stuck about how to make parentheses between arrow function argument, that is my only problem actually

File: stylesheet.css [hunk 1]
```diff
@@ -35,6 +35,41 @@ body {
   margin-right: 5%;
 }
 
+.m-navbar {
+  justify-content: space-between;
+  top: 0;
+  position: fixed;
+  height: 430px;
```
Comment (sja-thedude) [inline stylesheet.css]: - [ ] Kindly make sure to add the mobile menu height that covers the entire page as per the Figma design.

Figma design:

Screenshot 2022-05-09 at 11 41 15 PM

Your design:

https://user-images.githubusercontent.com/70269507/167494961-57665894-8f09-44e9-87a3-4d8366a41f34.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @mr-abdellah, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters are passing
✔️ Professional readme
✔️ Correct Gitflow
✔️ Great job in implementing the mobile menu correctly in the first attempt 👏 

## Required 
- [ ] Kindly update your readme with collaborator details and with steps to set up your project locally.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @mr-abdellah, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**